### ISSUE TITLE
Support new notification type for Wrapstodon

### ISSUE THREAD

**Issue by MagicLike** [tuskyapp/Tusky#4828]
### Pitch

Similar to #4334 with the new feature "Wrapstodon" (not on stable yet) a new notification type got released which currently only shows as `Unknown notification type` in Tusky.

### Motivation

As already described above it would actually show up correctly within Tusky and not as an error message

**Comment by connyduck** [comment]
I'm not sure if we will support this one. It is more of a gimmick than a real feature.

**Comment by MagicLike** [comment]
yeah and it is not even finished code-wise, just wanted to add an issue if it will be supported at some point and imo it is *very* low priority...
